src/sentry/static/sentry/app/components/organizations/timeRangeSelector/index.jsx

```diff
@@ -331,15 +331,24 @@ class TimeRangeSelector extends React.PureComponent {
                   )}
                 </SelectorList>
                 {isAbsoluteSelected && (
-                  <DateRange
-                    showTimePicker
-                    utc={this.state.utc}
-                    start={start}
-                    end={end}
-                    onChange={this.handleSelectDateRange}
-                    onChangeUtc={this.handleUseUtc}
-                    organization={organization}
-                  />
+                  <div>
```

Comment:
Is the div needed here?

Reply:
yes—it protects these two child divs from snapping into the flex layout and creating a three column layout instead of two. I could give it a name, but it needs no css, so a div is technically enough.